src/js/intl/strings.js

```diff
@@ -917,6 +917,12 @@ exports.strings = {
     'ru_RU': 'Спрятать цель'
   },
   ///////////////////////////////////////////////////////////////////////////
+  'objective-button': {
```

Comment:
yep youre right -- i guess i had never wrapped this in an intl call! I thought maybe the token was in this file and I just missed it in the transition